get.sh

```diff
@@ -233,7 +233,13 @@ getTestKitGenAndFunctionalTestMaterial()
 
 	mv openj9/test/TestConfig TestConfig
 	mv openj9/test/Utils Utils
-	mv openj9/test/functional functional
+    if [ -d functional ]; then
+        cd openj9/test/functional
+        find . -print | cpio -pdm ../../../functional
+        cd ../../..
+    else
+	    mv openj9/test/functional functional
+    fi
```

Comment:
Once we have `functional` in Adopt, this case will never happen.

Reply:
To keep directory structure, we should use rsync instead of cp or mv .
But AIX does not have rsync on default, so I selected cpio.